include/sisl/fds/buffer.hpp

```diff
@@ -74,6 +74,19 @@ struct sg_iterator {
         return ret_iovs;
     }
 
+    void move_offset(uint32_t const size) {
```

Comment:
This added functionality will need a MAJOR version bump, as will the additions below.

Reply:
True, will bump the major version.

Reply:
If you are going to bump the major version, it probably be fine. But if it turns out no, can you also do a cross build to make sure dependent packages (e.g. homestore/homerepl) builds fine?

Reply:
I see you already did the cross build, you can ignore my previous comment.

Reply:
Didn't we decide not to bump up version and rely on cross build all packages?

Reply:
I thought we decided the version _should_ be changed because the BUILD_NUMBER was not reliable. If that's the case, then the version should be changed to correctly identify the change made right? It's not like having big numbers is a problem as long as we don't have to keep changing it in the recipes right? When something _does_ depend on this change it can correctly set the minimum version that can be pulled that will reflect having this change made in the interface.